README.md

```
@@ -57,27 +162,43 @@ src/routes
 ||__ index.tsx
 ```
 
-#### Routes Created by TanStack Router
+`src/routeTree.gen.ts` is generated — do not edit it by hand. Run `pnpm routes:generate`
+after adding or renaming a route file.
+
+#### URLs these produce
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)